boefjes/boefjes/katalogus/configs.py

```diff
@@ -0,0 +1,27 @@
+import structlog
+from fastapi import APIRouter, Depends, Query
+
+from boefjes.dependencies.plugins import get_pagination_parameters
+from boefjes.models import BoefjeConfig, PaginationParameters
+from boefjes.sql.config_storage import get_config_storage
+from boefjes.storage.interfaces import ConfigStorage
+
+router = APIRouter(tags=["configs"])
+
+logger = structlog.get_logger(__name__)
+
+
+@router.get("/configs", response_model=list[BoefjeConfig])
+def list_configs(
+    organisation_id: str | None = Query(None),
+    boefje_id: str | None = Query(None),
+    enabled: bool | None = Query(None),
```

Comment:
You can leave out `Query` as FastAPI handles function arguments as query parameters by default (also to be consistent with the rest of the codebase)
```suggestion
    organisation_id: str | None = None,
    boefje_id: str | None = None,
    enabled: bool | None = None,
```

(`Query` can be handy for e.g. documentation or validation, but even then, with modern versions of FastAPI and Pydantic validation can also be done with (annotated) types)